samples/matter/closure/src/inc/ClosureControlEndpoint.h

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
```

Comment:
This is minor, but I would consider deleting inc directory, to be consistent with other samples. If you want to introduce some advanced structure of the files, I would recommend creating closures_control or similar directory and putting there all related .h and .*cpp files